community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/idmapping/string/ParallelSort.java

```diff
@@ -166,11 +164,11 @@ public ParallelSort( Radix radix, LongArray dataCache, long highestSetIndex,
         return rangeParams;
     }
 
-    private String dumpBuckets( int[][] rangeParams, int[] bucketRange, int[] bucketIndex )
+    private String dumpBuckets( long[][] rangeParams, int[] bucketRange, long[] bucketIndex )
```

Comment:
Another note for the future would be that we should probably pass a PrintStream in here, instead of building up a single humongous string.